src/ert/analysis/_es_update.py

```diff
@@ -515,15 +508,22 @@ def analysis_ES(
                 f"No active observations for update step: {update_step.name}."
             )
 
-        smoother = ies.ES()
+        smoother_es = ies.ESMDA(
```

Comment:
Need to pass in `inversion` here I think. It's `exact` by default.

Reply:
Thanks! Can you take a look if it seems correct?
From the IES library I see the following inversion types `"exact"` and `"subspace"`.
Looking at the `IES_INVERSION` keywords, it seems we have fore enumerated 0, 1, 2, 3 where 0 is exact and the three others are different forms of subspace.
I mapped them now, but should perhaps create an issue on changing the `IES_INVERSION` keyword and documentation? And perhaps EVEN resolve it in this PR? Is this a "breaking change" / changing behaviour for users thing?

https://github.com/tommyod/iterative_ensemble_smoother/blob/d544242a24f49537216a384e2af9c59c09a03739/src/iterative_ensemble_smoother/esmda.py#L174-L179

Reply:
In the IES `SIES` module I see
```python
    inversion_funcs = {
        "direct": inversion_direct,
        "subspace_exact": inversion_subspace_exact,
        "subspace_projected": inversion_subspace_projected,
    }
```
perhaps this is preferable?

Reply:
Some verification

In the SIES module I see the following inversion methods
```python
inversion_funcs = {
    "direct": inversion_direct,
    "subspace_exact": inversion_subspace_exact,
    "subspace_projected": inversion_subspace_projected,
}
```
The goal is to couple them to the **four** inversion types in ERT. From ERT docs I see
```
0: exact
1: subspace_exact_r
2: subspace_ee_r
3: subspace_re
```
In the old API of `SIES` I see
`_inversion_methods = ("naive", "exact", "exact_r", "subspace_re")`
which corresponds to the four types in ERT in that order.
We will change the inversion types `0,1,2,3 `in the ERT API later (note-to-self for issue). Therefore we now need to map them.

I think the following mapping of the ERT inversion types `0,1,2,3` to the new `inversion_funcs` in the new API is what is needed
```
0: "direct"
1: "subspace_exact"
2: "subspace_projected"
3: "subspace_projected"
```
I double checked the naming and descriptions in ERT and SIES with the four inversion types in the Evensen 2019 reference paper, to see that this makes sense.